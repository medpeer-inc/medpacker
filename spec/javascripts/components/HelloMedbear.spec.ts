import { mount } from '@vue/test-utils';
jest.mock('@image/medbear.png', () => 'test.png');
import Target from '@js/components/HelloMedbear.vue';

describe('@js/components/HelloMedbear', () => {
  it('is a Vue instance', () => {
    const wrapper = mount(Target);
    expect(wrapper.isVueInstance()).toBeTruthy();
    expect(wrapper.element).toMatchSnapshot();
  });
});
