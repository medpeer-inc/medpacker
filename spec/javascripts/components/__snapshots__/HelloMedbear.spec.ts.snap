// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@js/components/HelloMedbear is a Vue instance 1`] = `
<div>
  <div
    class="hello-medbear"
  />
   
  <img
    alt="Hello Medbear2"
    src="test.png"
  />
   
  <img
    alt="Hello Medbear3"
  />
</div>
`;
